_data-prepper/pipelines/configuration/processors/parse-json.md

```diff
@@ -30,6 +30,7 @@ This table is autogenerated. Do not edit it.
 | `overwrite_if_destination_exists` | No | Boolean | Whether to overwrite the destination field if it already exists. Default is `true`. |
 | `delete_source` | No | Boolean | Whether to delete the source field after parsing. Default is `false`. |
 | `tags_on_failure` | No | String | A list of tags to apply if parsing fails or an unexpected exception occurs. |
+| `handle_failed_events` | No | Enum | Determines how to handle events with JSON processing errors. Options include `skip`, which will log the error and send the event downstream to the next processor, `skip_silently`, which will send the event downstream to the next processor without logging the error, `drop` which will log the error and will not send the event downstream to the next processor, and `drop_silently`, which will not log error and will not send the event downstream to the next processor.|
```

Comment:
I think parse_json does not allow for the `drop` or `drop_silently` options (https://github.com/opensearch-project/data-prepper/blob/42faa8f34504a8cccdc9e511747b1151abaae0ed/data-prepper-plugins/parse-json-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/parse/json/ParseJsonProcessorConfig.java#L134)

Reply:
OK. Thanks for correcting